validation/src/main/scala/hmda/validation/engine/lar/quality/LarQualityEngine.scala

```diff
@@ -7,6 +7,11 @@ import hmda.validation.rules.lar.quality._
 
 trait LarQualityEngine extends LarCommonEngine with ValidationApi {
 
+  private def q022(lar: LoanApplicationRegister): LarValidation = {
+    // TODO: 2017 is hardcoded for now.  It will be passed in through the API at a later date
```

Comment:
The activity year is passed in from the front end.  From my understanding (and discussion with @jmarin), we currently have no actual data passing between the front- and back-end.  In order to have this quality edit pass, I have the year hard-coded.  Once we have a working connection, it should be simple to replace this hard-coded value with the value passed in from the front-end.